tests/kernel/mbox/mbox_api/src/test_mbox_api.c

```diff
@@ -433,6 +433,17 @@ void test_mbox_kdefine(void)
 	tmbox(&kmbox);
 }
 
+#if defined(CONFIG_CPU_ARCEM) || defined(CONFIG_CPU_CORTEX_M)
+/* For iotdk and ARC/nSIM, nSIM simulates full address space of memory,
+ * so all accesses outside of CCMs are valid and access to 0x0 address
+ * doesn't generate any exception. So skip this case.
+ */
+void test_mbox_data_get_null(void)
+{
+	ztest_test_skip();
```

Comment:
My suggestion is to write the #if-else statement into test_mbox_data_get_null() function definition, don't need to write them twice.